### DISCUSSION TITLE
what is verbosity ?

### DISCUSSION THREAD

**Discussion by jadbara** [ZakKemble/AVRDUDESS#66] [Q&A] [answered]
Hi, thank you for this great job !
I'm wondering what does "verbosity" do ?
thank you

**Comment by pcfreak1201** [comment, marked as answer]
It sets the amount of feedback/debug-information, getting from avrdude (->  -v -v -v -v  = 4) ... but don't use 4 if you really don't want to debug all kinds of code ... it takes a while ;-)
